modules/claims_api/spec/requests/v1/claims_request_spec.rb

```diff
@@ -166,34 +166,30 @@
 
       it 'shows a single errored Claim with an error message', run_at: 'Wed, 13 Dec 2017 03:28:23 GMT' do
         with_okta_user(scopes) do |auth_header|
-          create(:auto_established_claim,
+          create(:auto_established_claim, :status_errored_with_evss_response,
```

Comment:
for some reason, this test returns encounters a `404`, while the next one works just fine. The only difference I can see between the 2 is adding the EVSS error property to the mocked claim. Any ideas on how to fix this?

Reply:
this is the error i see when i step through the logic that test is testing:
```
#<NameError: undefined local variable or method `claim' for #<ClaimsApi::AutoEstablishedClaimSerializer:0x000055d76a2e6958>>
```